code/modules/events/blob.dm

```diff
@@ -8,6 +8,12 @@
 
 	gamemode_blacklist = list("blob") //Just in case a blob survives that long
 
+/datum/round_event/control/blob/canSpawnEvent(players, gamemode)
+	if(!SSshuttle.emergency || SSshuttle.emergency.mode != SHUTTLE_CALL || SSshuttle.canRecall()) // no blobs if the shuttle is past the point of no return
```

Comment:
Don't think this does what you intend it to. This will make the blob not start if the shuttle is idle, for example (not called at all).
This define is made for when it finally docks:
```suggestion
	if(EMERGENCY_AT_LEAST_DOCKED) // no blobs if the shuttle is past the point of no return
```
Maybe you could make a new one for when it's past the point of no recall?

Reply:
oop! good catch